src/replica/log_file.cpp

```diff
@@ -35,7 +35,8 @@ namespace dsn {
 namespace replication {
 
 log_file::~log_file() { close(); }
-/*static */ log_file_ptr log_file::open_read(const char *path, /*out*/ error_code &err)
+/*static */ log_file_ptr
+log_file::open_read(const char *path, /*out*/ error_code &err, size_t block_bytes)
```

Comment:
If you using the new FLAGS_XXX instead, then these parameters are not needed to pass everywhere.